src/ontology/foodon-edit.owl

```diff
@@ -123115,11 +123118,21 @@ Renamed from *FRUIT OR BERRY* in LanguaL 2008.</rdfs:comment>
 <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03420218">
 	<rdf:type rdf:resource="http://www.w3.org/2002/07/owl#Class"/>
 	<rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03420103"/>
-	<obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000428"/>
+	<obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000122"/>
 	<obo:IAO_0000115 xml:lang="en">Edible organ meat, including blood and bone marrow.</obo:IAO_0000115>
+	<obo:IAO_0000117 xml:lang="en">Rhiannon Cameron</obo:IAO_0000117>
 	<obo:IAO_0000412>http://langual.org</obo:IAO_0000412>
 	<oboInOwl:hasDbXref>http://www.langual.org/langual_thesaurus.asp?termid=C0218</oboInOwl:hasDbXref>
-	<oboInOwl:hasSynonym xml:lang="en">variety meat</oboInOwl:hasSynonym>
+	<oboInOwl:hasExactSynonym xml:lang="en">edible organs</oboInOwl:hasExactSynonym>
```

Comment:
I'm thinking it isn't necessary to add plurals unless they are irregular.  I've left an ontology curation slack note to discuss there.